mitmproxy/tools/console/master.py

```diff
@@ -123,12 +123,32 @@ def get_editor(self) -> str:
         else:
             return "vi"
 
+    def get_hex_editor(self) -> str:
+        if m := os.environ.get("MITMPROXY_EDITOR"):
+            return m
+        if m := os.environ.get("EDITOR"):
+            return m
+        for editor in "xxd", "ghex", "hexedit", "hexyl":
+            if shutil.which(editor):
+                return editor
+        if os.name == "nt":
+            for editor in "hexalter", "frhed", "HxD":
+                if shutil.which(editor):
+                    return editor
+                else:
+                    return "notepad"
```

Comment:
This code is clearly untested as it contains a major logic bug. Please don't rely on me catching your errors but test your code yourself. Ideally by writing tests. :)

Reply:
Okay, sir. I will add the required test and sorry for the inconvenience.